docs/docs-beta/docs/guides/migrate/version-migration.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
@neverett note that you requested `/guides/migration/version-migration` but this is in `/guides/migrate/version-migration` as the _migrate_ folder already existed.

Reply:
Whoops, yeah, I mistyped -- `/guides/migrate` is the correct folder. Thanks!